docs/device-management/5-ota/README.md

```diff
@@ -4,33 +4,75 @@ title: Over-the-Air (OTA) Updates Overview
+- [**CoAP Gateway Interface**](/reference/protocols/coap/ota)
 
-### Use Cases
+:::
 
-Here are some ideas on what can be created using our DFU Service.
+## Concepts
+
+### Artifacts
+
+An artifact is a binary image that can be pushed to your embedded device. This
+may take the form of the base level firmware (to be used with a bootloader), a
+binary blob for an external processor, a firmware image for the modem onboard
+(such as the nRF91), or a required content element for the running application,
+such as an image or sound file. Individual binaries (artifacts) are versioned
+and tracked before being added to a release.
+
+### Releases
+
+A release is a collection of artifacts to be delivered to the target embedded
+device. In the simplest example, a release will only contain a single artifact,
+the firmware image to reprogram the main processor on board. Once you mark a
+release as active, any eligible embedded devices will attempt to download and
+install that firmware update. Groups of devices may be targeted using Blueprints
+and Tags. Release rollout and rollback can be controlled from the [Golioth
+Console](https://console.golioth.io), the REST API, or via command line tools.
+
+### Blueprints
+
+A Blueprint is a way to indicate a unique hardware version for devices in your
+fleet. Blueprints help to segment which devices should be notified of a
+particular release. For instance, the release may support a different processor
+or different peripheral hardware for any given variation of fleet device.
+
+## Use Cases
+
+Here are some ideas on what can be created using our OTA Service.
 
 - Push new feature your devices in the field.
-  - Your product can be constantly evolving, so you can be constantly pushing new firmware securely to your fleet.
+  - Your product can be constantly evolving, so you can be constantly pushing
+    new firmware securely to your fleet.
 - Fix critical issues in your fleet of devices
   - Security patches, critical bug fixes, etc.
 - Edge computing scenarios
-  - A critical part on a edge computing scenario, is improving you Machine Learning model and updating devices with that latest optimized model.
-  - Push timeseries data to LightDB Stream, improve your ML model and upload it as an Artifact on our platform that can be part of a DFU Release.
+  - A critical part on a edge computing scenario, is improving you Machine
+    Learning model and updating devices with that latest optimized model.
+  - Push timeseries data to LightDB Stream, improve your ML model and upload it
+    as an Artifact on our platform that can be part of an OTA Release.
```

Comment:
This point confused me at first -- maybe this makes it more clear?
```suggestion
    as an Artifact on our platform that can be part of a subsequent OTA Release.
```

Reply:
@szczys any thoughts on this one?

Reply:
Got it, thanks for catching!